vars/createGithubIssue.groovy

```diff
@@ -64,7 +64,7 @@ def create_issue(component, url, currentVersion, failedMessage){
                 println("Issue already exists in the repository, skipping.")
             } else {
                 sh (
-                        script: "gh issue create --title \"[AUTOCUT] OS Distribution Build Failed for ${component}-${currentVersion}\" --body \"${message}\" --label ${label} --repo ${url}",
+                        script: "gh issue create --title \"[AUTOCUT] OS Distribution Build Failed for ${component}-${currentVersion}\" --body \"${message}\" --label ${label} --label \"untriaged\" --repo ${url}",
```

Comment:
Can we remove the ${label} since its just `autocut` hard code it and remove def from line 40